.circleci/config.yml

```diff
@@ -65,14 +66,33 @@ jobs:
     executor: cli
     steps:
       - checkout
+
+      - circle-compare-url/reconstruct
+
       - run:
           name: Publish dev releases of any modified orbs
           command: |
-            GIT_LOG=$(git log -1 --format="" --name-only)
+            # save value stored in file to a local env var
+            CIRCLE_COMPARE_URL=$(cat CIRCLE_COMPARE_URL)
```

Comment:
Super small question, but I found this line a little bit confusing at first glance. Would it be clearer if the file `CIRCLE_COMPARE_URL` looked more like a filename instead of a env var?

Reply:
ya that's a good suggestion @bradylill i can change it to `.txt`